packages/assistify-defaults/server/roles.js

```diff
@@ -189,6 +193,10 @@ const createConfigurationRole = function() {
 			'manage-own-integrations',
 			'manage-selected-settings'
 		]));
+
+	revokePermissions(CONFIGURATION_ROLE_NAME, [
+		'change-setting-Accounts_AllowedDomainsList'
```

Comment:
verified: this removes the permission if it was there for the manager earlier

Reply:
Is working. But when an admin is giving the right to `change-setting-Accounts_AllowedDomainsList` to the configExpert manually, it is gone after a server restart.
This might be intended - just for consideration